docs/crypto.rst

```diff
@@ -17,13 +17,13 @@ Operator                        Cost      Description
 :code:`Sha256(e)`               `35`      `SHA-256` hash function, produces 32 bytes
 :code:`Keccak256(e)`            `130`     `Keccak-256` hash funciton, produces 32 bytes
 :code:`Sha512_256(e)`           `45`      `SHA-512/256` hash function, produces 32 bytes
-:code:`Ed25519Verify(d, s, p)`  `1900`\*  `1` if :code:`s` is the signature of :code:`d` signed by private key :code:`p`, else `0`
+:code:`Ed25519Verify(d, s, p)`  `1900`\*  `1` if :code:`s` is the signature of :code:`d` signed by the private key pair of public key :code:`p`, else `0`
```

Comment:
Good find, maybe phrase as the private key corresponding to the public key P

Reply:
^ "private key sk corresponding to public key pk" suggested by Ben sounds more reasonable to end users